src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.split(';');
+  const styleObject = {};
```

Comment:
[CODE STYLE] - The variable name `styleObject` is too generic. Use a more descriptive name like `styleMap` or `cssProperties` to improve readability.